packages/fxa-payments-server/src/routes/Subscriptions/index.tsx

```diff
@@ -79,6 +79,11 @@ export const Subscriptions = ({
     }
   }, [ fetchSubscriptionsRouteResources, accessToken ]);
 
+  let showPaymentSuccessAlert = true;
```

Comment:
Should use a [`useState` hook](https://reactjs.org/docs/hooks-reference.html#usestate) to track this state, and a [`useCallback` hook](https://reactjs.org/docs/hooks-reference.html#usecallback) to wrap setting the state with `dismissAlert`. Otherwise, this will get set to `true` and the callback will be redefined every every time the component renders.